### DISCUSSION TITLE
Logging with NextJS, Axiom or Baselime ?

### DISCUSSION THREAD

**Discussion by pepew-le-boss** [vercel/next.js#65844] [Polls]
I'm trying to implement a logging system into my NextJS app and came across this page: [https://vercel.com/integrations#logging](https://vercel.com/integrations#logging).

After comparing the solutions, Axiom and Baselime seem the most interesting. Axiom appears more professional and is already used by many well-known companies. Baselime seems to be a newer company, but their pricing plan is very attractive (almost too good to be true).

Has anyone used either of these solutions and can provide some insights?

**Comment by haydenbleasel** [comment]
Personal opinion so feel free to debate or disregard. I gave Axiom a try initially, their free plan is very generous but wasn't super fond of the UX. Moved to [BetterStack](https://betterstack.com/), I'm happy with it. I haven't had a moment to test Baselime, but the founder reached out a while back and it looks great.

**Reply by JimmyLv** [reply]
while searching for the next.js logging system, I didn't expect to encounter Hayden again, 😂
